README.md

```diff
@@ -1,285 +1,33 @@
-<a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
 
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+ 
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
-- [📝 License](#license)
```

Comment:
- [ ] Good job using the right readme template for this project, as requested by the previous reviewer please edit the contents of your readme template to your project's details. Please note that instead of editing the `readme.md` file you deleted a majority of the sections. Please bring back the following sections in to your readme and make sure the contents are filled in.
  - About the project
  - Built With
  - Tech Stack
  - Getting started
  - Setup
  - Prerequisites
  - Install
  - Authors
  - Contributing
  - Show your supportt
  - Aknowledgements
  - License
- [ ] You should only remove a section of the readme file in your project only if it does not apply to your project as of now, otherwise please make sure it's filled in.
- [ Optional ] Still, on the readme file the link to the project LICENSE is not working, this is because your project does not have a file named LICENSE. Please add this file to the project and make sure its contents are those of the [MIT License](https://choosealicense.com/licenses/mit/) on this [link](https://choosealicense.com/licenses/mit/). Remember to customize the `year` and `fullname` fields of the License.